src/DIRAC/Resources/Computing/GlobusComputingElement.py

```diff
@@ -1,258 +0,0 @@
-########################################################################
-# File :   GlobusComputingElement.py
-# Author : A.S.
```

Comment:
I am OK with this getting dropped, but it must be mentioned explicitly in the release notes.